butler-bot/bot.js

```diff
@@ -63,4 +68,46 @@ client.on('message', message => {
     .then(() => { logger.info('Successfully added the role') })
 })
 
+client.setInterval((servers) => {
+  const itMuslimServer = servers.get(IT_MUSLIM_SERVER_ID)
+
+  const incompleteMembers = itMuslimServer.members.filter((member) => {
```

Comment:
Incomplete sounds strange. Like the JS objects are incomplete. It's better to call them as follows:
```suggestion
  const restrictedMembers = itMuslimServer.members.filter((member) => {
```